### DISCUSSION TITLE
Accidentally added duplicates, how to delete photos from command line?

### DISCUSSION THREAD

**Discussion by wscott** [immich-app/immich#3675] [Q&A]
So I imported my iphone photos that were saved with Nextcloud not realizing that many of these were still on my phone and that Immich won't notice the duplicates. So now I have duplicates in the library.
Nextcloud reencodes the HEIC files so the data isn't identical.
````
-rw-r--r-- 1 wscott wscott 2369923 Aug 13 07:51 '22-07-12 07-32-56 0888.jpg'
-rw-r--r-- 1 wscott wscott 1324940 Aug  3 08:28  IMG_0888.HEIC
````
Both files are most of the same EXIF information so I can identify the duplicates from the command line and I would hope in the future Immich can identify and remove these duplicates.  But for now, I would like to just identify the files and remove them.

Is there some way to remove files from the command line to tell Immich to rescan the directory and remove photos that have disappeared?

**Comment by agross** [comment]
You might want to have a look at this: https://github.com/agross/immich-duplicates

**Reply by wscott** [reply]
Nice! I will try it out.
